documentation/bolt_configuration_options.md

```diff
@@ -50,6 +50,7 @@ ssh:
 -   `proxyjump`: A jump host to proxy SSH connections through, and an optional user to connect with, for example: jump.example.com or user1@jump.example.com.
 -   `run-as`: A different user to run commands as after login.
 -   `run-as-command`: The command to elevate permissions. Bolt appends the user and command strings to the configured run as a command before running it on the target. This command must not require an interactive password prompt, and the `sudo-password` option is ignored when `run-as-command` is specified. The run-as command must be specified as an array.
+-   `sudo-executable`: The executable to use to run as another user when using `run-as`. **Note:** This option is experimental.
```

Comment:
I think it'd be useful to explain, either here or somewhere else in the docs, that this is in order to support using the executable with a password. You could use `run-as-command` for *most* of the functionality, except escalating with a password.

Reply:
`run-as` uses `sudo -S -u #{user} -p custom_prompt` then looks for the prompt to pass in the password if needed. `run-as-command` replaces that whole string with whatever you give it, and does not support using a password over stdin. `sudo-executable` only replaces the `sudo` part of the command, so would run `your_cmd -S -u #{user} -p custom_prompt`.